test/functional/feature_reindex.py

```diff
@@ -73,13 +73,36 @@ def find_block(b, start):
         # All blocks should be accepted and processed.
         assert_equal(self.nodes[0].getblockcount(), 12)
 
+    def continue_reindex_after_shutdown(self):
+        node = self.nodes[0]
+        self.generate(node, 1500)
+        self.log.info("Wait until block filter index is synced..")
+        with node.busy_wait_for_debug_log([b'basic block filter index is enabled at height']):
+            self.restart_node(0, ['-blockfilterindex'])
+
+        # Restart node with reindex and stop reindex as soon as it starts reindexing
+        self.log.info("Restarting node while reindexing..")
+        with node.busy_wait_for_debug_log([b'initload thread start'], timeout=3):
```

Comment:
thinking more about this, let's remove this timeout=3. It could fail on slow system or when running valgrind.